<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LT-1136" name="Sample(failed)" tests="1" failures="1" timestamp="7 Aug 2019 13:04:26 GMT" time="0.049" errors="0">
  <testcase name="Testing" time="0.024" classname="seleniumdemo5.FailedCases1">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at seleniumdemo5.FailedCases1.Testing(FailedCases1.java:10)
	at seleniumdemo5.Runner.main(Runner.java:17)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- Testing -->
</testsuite> <!-- Sample(failed) -->
